boltdesignsystem/bolt
# DS-501 Create Icon and Shape elements

## Jira

https://pegadigitalit.atlassian.net/browse/DS-501

## Summary

Introduces Icon and Shape elements that will replace Icon component.

## Details

1. Added Icon element. This only renders the inline SVG, nothing else.
2. Added Shape element. This is a 1:1 container that can be set to square or circle.
3. Icon and Shape elements together can replace all existing Icon component use cases.

## How to test

Run the branch locally and read through all the Icon and Shape element docs. Make sure all props work as expected.

## Release Notes
### Icon
The [Icon component](https://boltdesignsystem.com/pattern-lab/?p=viewall-components-icon) is deprecated. Use the Icon Element instead (todo: add link on release).
### Shape
Added the Shape Element (todo: add link on release).
